pkg/storage/zip.go

```diff
@@ -0,0 +1,9 @@
+package storage
+
+import "io"
+
+// Zip represents zip file of a specific version and it's size
+type Zip struct {
+	Reader io.ReadCloser
+	Size   int64
```

Comment:
Is the size actually *required* by Minio? What's the benefit of including size? I'd imagine That's something Minio should do itself once we give it a big blob. For example, I imagine S3/GCS calculate the size for you.